### PR TITLE
[CONV-43] API best practices

### PR DESCRIPTION
**Pull request by kacperk83**

### CODE REVIEW

**Comment by stevengunneweg** [comment]
All conventions are written in English, this one should be too to keep the conventions uniform

**Comment by kacperk83** [comment]
@stevengunneweg You are absolutely right. Will fix that (the text has been copied from Confluence).

File: php/api/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+##Endpoint design
+
+**Never use verbs and always use plural nouns** 
+
+**The right HTTP method** 
+
+**Use query parameters for more specific queries**
+
+Wrong:
```
Comment (stevengunneweg) [inline php/api/README.md]: Please keep right/wrong blocks inline with other conventions/pages

File: php/api/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+/v1/productByName
+/v1/productsByClient
+```
+Better:
+```php
+GET /v1/products
+```
+
+```php
+GET /v1/products/1?name=
```
Comment (stevengunneweg) [inline php/api/README.md]: this can be added to the 'better' codeblock above

File: php/api/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+```php
+GET /v1/products?client=
```
Comment (stevengunneweg) [inline php/api/README.md]: this can be added to the 'better' codeblock above

**Review by stevengunneweg — CHANGES_REQUESTED** [review]

File: php/api/README.md (hunk 1 above)
Comment (stevengunneweg) [reply php/api/README.md]: Right:
```
<code>
```

Wrong:
```
<code>
```

**Review by stevengunneweg — APPROVED** [review]

**Review by edgargressie — CHANGES_REQUESTED** [review]
Please have a look at the markdown syntax. When switching to the visual/rendered view you will see some mistakes. The titles, the list beneath 'Ednpoint design' and the strong 'right' and 'wrong' headings for example. See other conventions for the right syntax.

File: api/README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+## Always use HTTPS
+
+## Endpoint design
+
+**Never use verbs and always use plural nouns** 
+
+**The right HTTP method** 
+
+**Use query parameters for more specific queries**
```
Comment (bartvandebiezen) [inline api/README.md:17]: This is an action/rule. Should be a heading.

**Pull request closed without merging**